packages/core/test/test-util/test-services/v4/test-service-temporal/TestEntityRequestBuilder.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved.
+ *
+ * This is a generated file powered by the SAP Cloud SDK for JavaScript.
+ */
+import {
+  RequestBuilder,
+  GetAllRequestBuilderV4,
+  EntityV4, Constructable, DestinationOptions, DestinationNameAndJwt, Destination, DTMiddlewareInterface
+} from '../../../../../src';
+import { TestEntity as TestEntityTemporal, TestEntity } from './TestEntity';
+
+/**
+ * Request builder class for operations supported on the [[TestEntity]] entity.
+ */
+export class TestEntityRequestBuilder<T1 = string, T2 = number> extends RequestBuilder<TestEntity<T1, T2>> {
+  // A new class is needed to solve the nested generic type
```

Comment:
I think you should be able to do this in the requestbuilder directly, when you resolve the generic type there.